### PR TITLE
Docs: update debug.md

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <long0dai@foxmail.com>

* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Update debug.md.

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Comment by daixiang0** [comment]
ping @fisherxu @kevin-wangzefeng

File: content/en/docs/advanced/debug.md [hunk 1]
```diff
@@ -3,9 +3,113 @@ draft: false
...
+    ```
+    (Warning: the same **terminal** is essential to continue the work, or it is necessary to type this command again.) Checking the environment variable with the following command:
+    ``` shell
+    echo $CLOUDCOREIPS
+    ```
+
+3. Generate the certificates for **CloudStream** on cloud node, however, the generation file is not in the `/etc/kubeedge/`, we need to copy it from the repository which was git cloned from GitHub.
+   Change user to root:
+    ```shell
+    sudo su
```
Comment (kevin-wangzefeng) [inline content/en/docs/advanced/debug.md:34]: What would happen if user don't use root?

Comment (daixiang0) [reply content/en/docs/advanced/debug.md:34]: In this case, all cmds should start with `sudo` first.

**Pull request merged**